src/Main.java

```diff
@@ -0,0 +1,22 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+
+	public static void main(String[] args) throws Exception {
+		System.out.println(makeListAndSortByAge("Persons.csv"));
+	}
+	public static List<Person> makeListAndSortByAge(String fileName) throws Exception{
+		BufferedReader br = new BufferedReader(new FileReader(fileName));
+		return br
+				.lines()
+				.map(s -> s.split(","))
+				.filter(a -> Integer.parseInt(a[2].trim()) >17)
```

Comment:
This lambdas can be encapsulated in a separate method and then you just pass the method reference to the `filter()` and `map()`